go/vt/log/log.go

```diff
@@ -78,5 +81,32 @@ var (
 // calls this function, or call this function directly before parsing
 // command-line arguments.
 func RegisterFlags(fs *pflag.FlagSet) {
-	fs.Uint64Var(&glog.MaxSize, "log_rotate_max_size", glog.MaxSize, "size in bytes at which logs are rotated (glog.MaxSize)")
+	flagVal := logRotateMaxSize{
+		val: "1887436800", // glog.MaxSize value (which is not concurrency safe)
```

Comment:
Instead of copying the value here, can't we use `glob.MaxSize` itself here? Maybe with an atomic load then I guess?

Reply:
oh, lol, come on GitHub PR UI https://github.com/vitessio/vitess/pull/12546#discussion_r1125686188

so, uhh, yeah I agree with @dbussink here 😂

Reply:
That was the first thing I tried and it didn't help.

Reply:
Giving it a go in the CI now (will do 20 test runs again): https://github.com/vitessio/vitess/pull/12546/commits/0c5be06fa97184e86118c9086fe4cf75e9de4bd5

IIRC, the race moved to pflag.Parse/TryParse around flag.TrickGlog but let's see.